android-core/src/main/java/com/mparticle/MParticleOptions.java

```diff
@@ -421,6 +434,18 @@ public Builder installType(@NonNull MParticle.InstallType installType) {
             return this;
         }
 
+        /**
+         * Add locsideloadedKitsal kits
```

Comment:
```suggestion
         * Add sideloaded kits
```

Looks like a copy/paste error here?

Reply:
Yup, Ill fix it.